index.html

```diff
@@ -0,0 +1,161 @@
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#" class="campaign">Campaign</a></li>
+          </ul>
+        </div>    
+      </div>
+    </nav>
+  </section>
+  
+  
+  <section class="main">
+    <header>
+      <nav class="nav">
+        <div class="top-links">
+          <a class="top-link" href="/about.html">About</a>
+          <a class="top-link" href="#">program</a>
+          <a class="top-link" href="#">Join</a>
+          <a class="top-link" href="#">Sponsor</a>
+          <a class="top-link" href="#">News</a>
+        </div>
+        <div class="hamburger" id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <div class="main-bg absolute"></div>
+    <div class="main-text">
+      <h1>Welcome, English learners.</h1>
+      <h class="green-text">Join the PCC to conquer public speaking forever.</h>
+    </div>
+    <div class="scnd-text">
+      <p>
+        Join over three hundred students to learn and practice presentation skills and public
+        speaking. Compete aganist your colleage and win prestigious prizes.
+      </p>
+    </div>
+    <h2>2024.02.04 ~ 28</h2>
+    <span class="EventLocation">@ ThecEnglish Club, Alexandria, Egypt</span>
+  </section>
+
+  <section class="program-container">
+    <h1>Main Program</h1>
+    <div class="mid">
+      <hr class="line"/>
+    </div>
+  <div class="cards-div">
+  <div class="card">
+    <img src="./images/about.png" alt="Icon of About IU">        
+    <h4>About PCC</h4>
+    <p>
+        PCC stands for "Presentation Course and Competition"
+    </p>
+  </div>
+  <div class="card">
+    <img src="./images/studyprograms.png" alt="Icon of Study programmes at IU">        
+    <h4>Program</h4>
+    <p>
+        Choose your trainer. Compete aganist other teams.
+    </p>
+  </div>
+  <div class="card">
+    <img src="./images/studyingAtIU.png" alt="Icon ">        
+    <h4>Career</h4>
+    <p>
+        Learn competent career bulding skills.
+    </p>
+  </div>
+  <div class="card">
+    <img src="./images/scolarship.png" alt="Icon of Scholarship at IU">        
+    <h4>Certificate</h4>
+    <p>
+        Earn certificate of completion. 
+    </p>
+  </div>
+  <div class="card">
+    <img src="./images/qustions-answers.png" alt="Icon of Q&A session">        
+    <h4>Q&A session</h4>
+    <p>
+        Do you need help with your decision? Your personal student advisor is there!
+    </div>
+</div>
+  <button class="register" type="button">Register for the event</button>
+  <p class="see-program">See Whole Program</p>
+</section>
+
+<section class="speakers">
+  <h3>Featured Speakers</h3>
+  <div class="mid">
+    <hr class="line"/>
+  </div>
+  <div class="list"></div>
+  <div class="more-btn">
+      <span>MORE</span> 
+      <i class="fas fa-chevron-down"></i>
+  </div>
+</section>
```

Comment:
**Fantastic work on dynamically generating your featured speakers using JavaScript!** 👏 I have some **styling suggestions** to help you achieve a closer resemblance to the provided design:

- [ ] Please ensure that you are using the exact colors specified in the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) throughout your entire project. **Consistency in color application will significantly enhance the overall visual coherence.** Kindly update the colors to match the following:
   ```
   Colors: #272A31, #EC5242, #D3D3D3
    ```

- [ ] It would be great if you could **align the separator line between the name and the description** of each featured speaker. Ensuring proper alignment will **add a professional touch and make the section more visually appealing.** 🔍

   **_Implementing these suggestions will help your project align closely with the provided design. Keep up the amazing work! 🚀_**

   ### Your design:
   ![image](https://github.com/mohamedgamaleldin999999/Capstone-project-module-one/assets/107952724/38167548-c7c0-49d3-8781-0b78d9e4986b)

   ### Behance design:
   ![image](https://github.com/mohamedgamaleldin999999/Capstone-project-module-one/assets/107952724/7c01050d-ffc9-4240-8ad2-314c8bd578ee)
   
   _Just a friendly reminder that adhering as closely as possible to the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) is a requirement for this project. While you've done a great job so far, it's important to keep the template in mind as you continue with your design._ 🙋‍♂️
   ![image](https://github.com/mohamedgamaleldin999999/Capstone-project-module-one/assets/107952724/8f81b362-9ff7-4429-9cc4-df89384affe0)